source/js/kitchen.js

```diff
@@ -146,24 +162,35 @@ function loadTemplateFromURL(templateURL, callback) {
     });
 }
 
+function presenterForType(type) {
+    console.log(type);
```

Comment:
Could you remove this `console.log` ?
I don't think this prints anything anyways.